app/page/channel.js

```diff
@@ -39,7 +39,9 @@ exports.create = function (api) {
       when(subscribed, 'Unsubscribe from channel', 'Subscribe to channel')
     )
 
-    const composer = api.message.html.compose({ meta: { type: 'post', channel: channelName } })
+    const composer = api.message.html.compose({
+      location: { page: '/channel' },
```

Comment:
note we can and should just use the location on line 22 - it has the correct channel name built in.